ext/reflection/php_reflection.c

```diff
@@ -2642,23 +2718,33 @@ ZEND_METHOD(reflection_parameter, isDefaultValueAvailable)
 ZEND_METHOD(reflection_parameter, getDefaultValue)
 {
 	parameter_reference *param;
-	zend_op *precv;
 
 	if (zend_parse_parameters_none() == FAILURE) {
 		RETURN_THROWS();
 	}
 
 	param = _reflection_param_get_default_param(INTERNAL_FUNCTION_PARAM_PASSTHRU);
 	if (!param) {
-		return;
+		RETURN_THROWS();
 	}
 
-	precv = _reflection_param_get_default_precv(INTERNAL_FUNCTION_PARAM_PASSTHRU, param);
-	if (!precv) {
-		return;
+	if (param->fptr->type == ZEND_INTERNAL_FUNCTION) {
+		zval default_value_zval;
+		if (_reflection_param_get_default_arg_info(param->fptr, (zend_internal_arg_info*) (param->arg_info), &default_value_zval) == FAILURE) {
+			RETURN_THROWS();
+		}
+
+		ZVAL_COPY(return_value, &default_value_zval);
+		zval_dtor(&default_value_zval);
```

Comment:
ZVAL_COPY + zval_dtor doesn't make a lot of sense, you can use ZVAL_COPY_VALUE instead.